README.md

```diff
@@ -234,3 +234,38 @@ git clean -xffd && git submodule foreach --recursive git clean -xffd
 ```
 
 Then, follow all the previous steps from the beginning.
+
+## RESEARCH
+
+## Buffer Pool
+
+Buffer pool is an in-memory cache used for faster access to frequently accessed data. It stores data and indexes reducing disk I/O.
+
+### Working of Buffer Pool
+
+Buffer pool mechanism works on the the two sublist concepts the **new sublist** and the **old sublist** where every item when accessed first time gets on the top of old sublist and on being called while in the old sublist moves to the top of new sublist. By default **37%** of space is reserved for old-block.
```

Comment:
What is the mechanism from moving tuples out of the new sublist? is it LRU, does tuples going out of the new sublist go back into the old sublist?

Reply:
The mechanism of moving tuples out of the new sublist and old sublist is same as both of them works on the LRU algorithm. Once the block moves from old sublist to the head of new sublist then all the previously present blocks in new sublist move one step below and if the new sublist reaches its capacity then the least recently used block from new sublist moves to the head of old sublist.

Reply:
okay, add that to your pr and I will merge it.

Reply:
I have added the relevant explanation in my new commit, you can review and merge the PR.

Reply:
@JoshInnis  I've updated the explanation in readme as you instructed so please merge my PR.